Kconfig

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2023 Renesas Electronics Corporation.
+# Copyright (C) 2023 EPAM Systems, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+config DOMD_ID
+	int "Dom D id"
+	default 1
+
+config SM_VCHAN_PATH
```

Comment:
It is not only SM will use this path. Will we create another vchan for IAM for example?

Reply:
yes it will be another vchan for iam